app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+const SupplementalApplicationContainer = ({}) => {
+  return (
+    <React.Fragment>
```

Comment:
should we move the status-comment section to its own component?

Reply:
yes, I put it just because I needed some content, so the container does not break. However, it was pretty annoying seeing there, so I made it component in the status history feature branch. https://github.com/Exygy/sf-dahlia-lap/pull/25/files#diff-9f722a1dce071f125d671dc5f6ee03e7R8 It needs some cleanup, but we could take care when we implement that story. Right now it's just a dump of html.

Reply:
:+1: